DPPP/Filter.cc

```diff
@@ -265,12 +265,12 @@ namespace DP3 {
       // Remove these rows from the ANTENNA table.
       // Note that stations of baselines that have been filtered out before,
       // will also be removed.
-      Vector<uInt> removedAnt = selTab.rowNumbers();
-      Vector<Int> antMap = createIdMap (antTab.nrow(), removedAnt);
+      Vector<rownr_t> removedAnt = selTab.rowNumbers();
```

Comment:
While this is a `rownr_t`, it maps to the antenna table, I think we can safely assume the number of antennas to fit in an int.